test/remote_SUITE.erl

```diff
@@ -272,35 +278,28 @@ multiple_casts_test(_Config) ->
                              ),
            ok
        end,
-       fun(_Ret, Trace) ->
-               %% 1. Check that all casts were received by a local client process before sending over net:
-               ?assert(
-                  ?strict_causality( #{?snk_kind := test_cast, seqno := _SeqNo}
-                                   , #{?snk_kind := gen_rpc_cast, cast := {cast, gen_rpc_test_helper, test_call, [_SeqNo]}}
-                                   , Trace
-                                   )),
-               ?assert(
-                  ?strict_causality( #{?snk_kind := gen_rpc_cast,      cast   := _Cast, sendto := _SendTo}
-                                   , #{?snk_kind := gen_rpc_send_cast, packet := _Cast, sendto := _SendTo}
-                                   , Trace
-                                   )),
-               %% 2. Check that no message reordering occurs on the client side:
-               snabbkaffe:strictly_increasing(?projection(packet, ?of_kind(gen_rpc_send_cast, Trace))),
-               %% 3. Check that no errors were detected:
-               ?assertMatch([], ?of_kind(gen_rpc_error, Trace)),
-               %% 4. Check delivery of the messages:
-               ?assert(
-                  ?strict_causality( #{?snk_kind := gen_rpc_send_cast, packet := _Packet}
-                                   , #{?snk_kind := gen_rpc_acceptor_receive, packet := _Packet}
-                                   , Trace
-                                   )),
-               %% 5. Check that all the messages were delivered:
-               Calls = ?of_kind(do_test_call, Trace),
-               ?projection_complete(seqno, Calls, L),
-               %% 6. Check that all calls are executed in order:
-               %% snabbkaffe:strictly_increasing(Calls), %% FIXME: Fails due to bug
```

Comment:
Q: would it make sense to keep this commented in the `test/gen_rpc_trace_props.erl` to avoid forgetting this property?  It seems that this is _almost_ the same as the property in `multiple_ordered_casts_test`, except that this uses `?projection_complete`.

Reply:
This check is actually implemented here: https://github.com/emqx/gen_rpc/pull/22/files#diff-a06c361dfcd997d65165fb44bc499743fec83d5e8be25ffaae45a421ef6e5e6bR300
So the TODO is done.